src/idl_gen_grpc.cpp

```diff
@@ -65,10 +67,15 @@ class FlatBufMethod : public grpc_generator::Method {
   std::string get_input_type_name() const {
     return (*method_->request).name;
   }
+
   std::string get_output_type_name() const {
     return (*method_->response).name;
   }
 
+  std::string get_fb_builder() const {
+    return "builder";
```

Comment:
why is this needed?

Reply:
To access the `SerializeToString` and `FromString` function from flatbuffers. Just like we do in `CreateString`.

Reply:
I don't understand why this has to be in a method though. What other values can this method return? Why is it relying on the name of the builder variable?